### PR TITLE
Various changes in measure theory chapter

### PR DESCRIPTION
**Pull request by user202729**

### CODE REVIEW

File: tex/homology/singular.tex [hunk 1]
```diff
@@ -666,7 +666,7 @@ \section{More examples of chain complexes}
 	$[v_0,\ldots,v_n]$, we remove each vertex one-by-one, and take
 	the alternating sum. Therefore, $\partial([v]) = []$. Extending
 	it linearly to complexes yields
-	$\partial(\sum n_ip_i) = \sum n_i \cdot 1$ --- so $\epsilon$ really is
+	$\partial(\sum n_ip_i) = \sum n_i \cdot 1$ --- so $\eps$ really is
```
Comment (user202729) [inline tex/homology/singular.tex:669]: For consistency --- everywhere else in the book either \eps or \varepsilon is used (which renders the same)

File: tex/measure/lebesgue-int.tex [hunk 2]
```diff
@@ -80,10 +87,25 @@ \section{The definition}
 	The constant function $1$ is \emph{not} integrable on $\RR$,
 	since $\int_\RR 1 \; d\mu = \mu(\RR) = +\infty$.
 \end{example}
-For this reason, I will usually prefer the term ``integrable''.
+For this reason, I will usually prefer the term ``absolutely integrable''.
```
Comment (user202729) [inline tex/measure/lebesgue-int.tex:90]: surely this is what you mean right (you've just complained ``integrable'' is confusing terminology above)

File: tex/measure/lebesgue-int.tex [hunk 3]
```diff
@@ -98,13 +120,21 @@ \section{The definition}
...
 		- \int_\Omega |f^-| \; d\mu \]
 	which in particular is finite.
 \end{step}
+
+That said, calling it ``finitely integrable'' here would also make it as easy to remember:
+\begin{exercise}
+	Show that $\int_\Omega |f| d\mu < \infty$ if and only if
+	$\int_\Omega |f^+| d\mu < \infty$ and
+	$\int_\Omega |f^-| d\mu < \infty$.
+\end{exercise}
```
Comment (user202729) [inline tex/measure/lebesgue-int.tex:136]: mnemonic is cool, but doesn't help much with understanding.

File: tex/measure/lebesgue-int.tex [hunk 4]
```diff
@@ -147,6 +177,69 @@ \section{The definition}
 as we won't really have a use for them in the context of probability.
 (They appear later on in a bonus chapter.)
 
+\section{An equivalent definition}
```
Comment (user202729) [inline tex/measure/lebesgue-int.tex:180]: approach used from Wheeden. In chapter 1-9 of that textbook before ``abstract integration'', the only measure space that is used is ℝ^n, which means the measure over Ω × ℝ is obtained ``for free''.

This definition also allows proving the monotone convergence theorem really easily.

File: tex/measure/lebesgue-int.tex [hunk 5]
```diff
@@ -147,6 +177,69 @@ \section{The definition}
...
+\end{moral}
+
+The last step is exactly the same as in the previous section.
+\begin{step}
+	[Absolutely integrable functions]
+	If $f \colon \Omega \to [-\infty, \infty]$ is absolutely integrable,
+	then we define
+	\[ \int_\Omega f \; d\mu = \int_\Omega |f^+| \; d\mu
+		- \int_\Omega |f^-| \; d\mu. \]
+\end{step}
```
Comment (user202729) [inline tex/measure/lebesgue-int.tex:241]: Well, proof of equivalence is "conveniently" omitted...

File: tex/measure/lebesgue-int.tex [hunk 6]
```diff
@@ -157,6 +250,18 @@ \section{Relation to Riemann integrals (or: actually computing Lebesgue integral
...
 
+Note that a Riemann integrable function is \emph{bounded},
+which means if you try to construct a function $f \colon [0, 1] \to \RR$
+in the same vein as \Cref{prob:sin_improper} by
+\[
+	f(x) = \begin{cases}
+		\frac{\sin(1/x)}{x} & x > 0 \\
+		0 & x = 0
+	\end{cases}
+\]
```
Comment (user202729) [inline tex/measure/lebesgue-int.tex:261]: "common mistake"? (not sure how many people would make this mistake...)

File: tex/measure/measure-space.tex [hunk 7]
```diff
@@ -351,17 +354,54 @@ \section{A hint of Banach-Tarski}
...
+only if $S$ is a measurable set.}
 In the past, when we had topological spaces,
 we considered continuous functions.
 The analog here is:
 \begin{definition}
 	Let $(X, \SA)$ and $(Y, \SB)$ be measurable spaces
 	(or measure spaces).
 	A function $f \colon X \to Y$ is \vocab{measurable}
-	if for any measurable set $S \subseteq Y$ (i.e.\ $S \in \SB$)
+	if for any open set $S \subseteq Y$ (i.e.\ $S \in \SB$)
```
Comment (user202729) [inline tex/measure/measure-space.tex:367]: scary mistake. (although for the Borel measure, they're conveniently the same thing)

File: tex/measure/measure-space.tex [hunk 8]
```diff
@@ -351,17 +354,54 @@ \section{A hint of Banach-Tarski}
...
 	we have $f\pre(S)$ is measurable (i.e. $f\pre(S) \in \SA$).
 \end{definition}
 
+Apart from the obvious symmetry with the definition of continuous function,
+as we will see in \Cref{sec:lebesgue_int_equivalent_def}, this definition is such that
+for a nonnegative function $f \colon \Omega \to \RR_{\geq 0}$,
+$\int_\Omega f \; d\mu$ exists if and only if $f$ is measurable.
```
Comment (user202729) [inline tex/measure/measure-space.tex:374]: motivate the definition a bit, hopefully.

File: tex/measure/swapsum.tex [hunk 9]
```diff
@@ -43,20 +43,65 @@ \section{Motivating limit interchange}
...
-			1 & x = 0 \\
-			0 & \text{otherwise}.
-		\end{cases} \]
-\end{exercise}
-This is why when thinking about the Riemann integral
-it is commonplace to work with stronger conditions like
-``uniformly convergent'' and the like.
-However, with the Lebesgue integral, we can mostly not think about these!
+
+By ``the Lebesgue integral has better behavior'', we means the following:
```
Comment (user202729) [inline tex/measure/swapsum.tex:47]: I'm... pretty sure before the change, the book misses the *point* of the convergence theorems.

The "better behavior" described in this section is actually a feature of *measurable* functions (i.e. lim sup and lim inf of a sequence of measurable functions is still measurable, this property is not hold for continuous functions or Riemann integrable functions)

**Comment by user202729** [comment]
Speaking of which, do you have some configuration to indent the `\ii` in `\begin{itemize}` environment? Because when I use `\ii` instead of `\item`, when I close the environment vimtex will end up putting the `\end{itemize}` one indent less than the `\begin{itemize}` line.

File: tex/measure/swapsum.tex [hunk 10]
```diff
@@ -208,6 +293,67 @@ \section{Everything else}
...
+
+In other words,
+\begin{moral}
+	The fact that $\int f \; d\mu \neq \lim \int f_k \; d\mu$ is only caused by
+	$\int_{ \Omega \setminus U } f \; d\mu \neq \lim \int_{ \Omega \setminus U } f \; d\mu$.
+\end{moral}
+
+\begin{example}
+	[Removing a set of small measure will allow interchanging the integral and the limit]
+	We take a few examples from \label{ex:failure_interchange_lim_int}, and see what happens if we remove
```
Comment (user202729) [inline tex/measure/swapsum.tex:336]: I definitely meant to write \Cref here... will fix soon.

File: tex/measure/lebesgue-int.tex (hunk 2 above)
Comment (vEnhance) [reply tex/measure/lebesgue-int.tex:90]: yep!

File: tex/measure/caratheodory.tex [hunk 11]
```diff
@@ -455,7 +456,8 @@ \section{Defining the Lebesgue measure}
 there are more Lebesgue measurable sets than Borel ones.
 Indeed, it can actually be proven using transfinite induction
 (though we won't) that
-$\left\lvert \SB(\RR) \right\rvert = \left\lvert \RR \right\rvert$.
+$\left\lvert \SB(\RR) \right\rvert = \left\lvert \RR \right\rvert$.\footnote{
```
Comment (vEnhance) [inline tex/measure/caratheodory.tex:459]: i think this leaves an undesirable space at start of footnote, i'll fix it

File: tex/measure/measure-space.tex (hunk 7 above)
Comment (vEnhance) [reply tex/measure/measure-space.tex:367]: I'm a bit confused by this. Is "measurable" not right?

https://en.wikipedia.org/wiki/Measurable_function

File: tex/measure/swapsum.tex (hunk 9 above)
Comment (vEnhance) [reply tex/measure/swapsum.tex:47]: sounds correct.

File: tex/measure/lebesgue-int.tex (hunk 6 above)
Comment (vEnhance) [reply tex/measure/lebesgue-int.tex:261]: Maybe it's clearer to write "Riemann integrable function _must be_ bounded"?

File: tex/measure/measure-space.tex [hunk 12]
```diff
@@ -170,7 +170,10 @@ \section{$\sigma$-algebras and measurable spaces}
 (which basically turns out to be impossible),
 we just give up and say ``$\SA$ is all the sets you can get if you start
 with the open sets and apply repeatedly union/complement operations''.
-Or even more bluntly: ``start with open sets, shake vigorously''.
+Or even more bluntly: ``start with open sets, shake vigorously''.\footnote{
```
Comment (vEnhance) [inline tex/measure/measure-space.tex:173]: note to self to fix space here

File: tex/measure/lebesgue-int.tex [hunk 13]
```diff
@@ -80,10 +87,25 @@ \section{The definition}
...
+		\inf_{0 \le s \le f} \left( \int_\Omega s \; d\mu \right) \]
+	where the infimum is taken over all simple $s$ such that $f \leq s$.
+	However, if the functions are unbounded or $|\Omega| = \infty$, the situation is not that simple:
+	\begin{itemize}
+		\ii Show that $f(x) = x^{-2}$ over $\Omega = (1, \infty)$ is absolutely integrable,
+		yet for all simple $s$ such that $f \leq s$, then $\int_\Omega f \; d\mu = \infty$.
+		\ii Show that $f(x) = x^{-0.5}$ over $\Omega = (0, 1)$ is absolutely integrable,
+		yet there's no simple $s$ such that $f \leq s$ and $s$ is finite almost everywhere.
+	\end{itemize}
+\end{remark}
```
Comment (vEnhance) [inline tex/measure/lebesgue-int.tex:107]: maybe this should be "exercise" and not "remark" if the imperative is "show that"

**Review by vEnhance — CHANGES_REQUESTED** [review]

File: tex/measure/measure-space.tex (hunk 7 above)
Comment (user202729) [reply tex/measure/measure-space.tex:367]: For Borel measurable function it's the same, for Lebesgue measurable function it's not. I add two boxes below to explain it.

**Comment by user202729** [comment]
Regarding the extra space in footnote: my oversight yes, but looks like LaTeX has an appropriate `\ignorespaces` to guard against it.

image

Not that it matters.

File: tex/measure/caratheodory.tex [hunk 14]
```diff
@@ -505,9 +507,9 @@ \section{A fourth row: Carath\'{e}odory for pre-measures}
 to $\mu$ on $\SA$, where $\SA$ is the $\sigma$-algebra generated by $\SA_0$.
 Indeed, our original goal was to get $\SB(\RR^n)$, and in fact:
 \begin{exercise}
+	\label{exer:cubes_vs_open}
```
Comment (user202729) [inline tex/measure/caratheodory.tex:510]: Apparently, the other way (without a `%` on the preceding line) introduces an unwanted space. (on my machine)

Before/after:

image

Comment (vEnhance) [reply tex/measure/caratheodory.tex:510]: Good catch.

**Review by vEnhance — APPROVED** [review]

**Pull request merged**